_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,95 @@
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6039 -->
+* Provide intention to create a method from an unresolved method call (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/create_method" w="364px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5928 -->
+* Add initial parsing of the new [asm! macro](https://github.com/rust-lang/rfcs/blob/master/text/2873-inline-asm.md) syntax (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5720 -->
+* Supersede `Fill match arms` intention with `Add remaining patterns` quick fix which supports empty `match` expressions now
+
+    {% include gif-img.html path="/assets/posts/changelog-131/empty_match_add_remaining_patterns" w="350px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5369 -->
+* Place methods with `&mut self` at the end of completion list for non-mutable variables
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5657 -->
+* Handle name conflicts after renaming local variables (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5956 -->
+* Add `dbgr` postfix completion (by [@mgrachev])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/dbgr" w="406px" %}
+
+* Improvements of `Move` refactoring for top-level items (`Refactor | Move` or <kbd>F6</kbd>):
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6010 -->
+    * Make moved items public if needed
+
+        {% include gif-img.html path="/assets/posts/changelog-131/move_make_items_public" w="373px" %}
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6078 -->
+    * Support finding visibility conflicts in more cases
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6057 -->
+* Introduce a quick fix to add `..` to struct pattern that has missing fields (by [@t-kameyama])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/pat_rest" w="400px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5995 -->
+* Rename "Cargo Command" run configuration to "Cargo"
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6075 -->
+* Fix "File is not included in module tree" warnings for a project under a symlinked directory
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6033 -->
+* Don't resolve macros from `std` with `no_std` attribute (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6041 -->
+* Fix highlighting inside macro calls in some cases
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6049 -->
+* Fix flattening nested `use` statements intention (by [@t-kameyama])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6056 -->
+* Fix benches flag in `cargo` command completion
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6102 -->
+<!-- * INT: add missing title and fix typo in CreateFunctionIntention (by [@Kobzol]) -->
```

Comment:
Let's drop commented items - I dropped them from this milestone

Reply:
I dropped commented items not included in the milestone, but this commit is included in the milestone

Reply:
Oh, we usually attach all links to the corresponding item
Something like
```
<!-- https://github.com/intellij-rust/intellij-rust/pull/N1 -->
<!-- https://github.com/intellij-rust/intellij-rust/pull/N2 -->
* Some useful text
```
See `130` changelog as an example

Reply:
Thanks, I forgot about this